functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,39 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals isPalindrome*/
+
+describe("Checking if word is palindrome: ", function () {
+    it("word tenet is a palindrome", function () {
+        var resultEntry = isPalindrome("tenet");
+        expect(resultEntry).toBeTruthy();
+    });
+    it("word dog is a palindrome", function () {
+        var resultEntry = isPalindrome("dog");
+        expect(resultEntry).toBeFalsy();
+    });
+     it("word kayak is a palindrome", function () {
+        var resultEntry = isPalindrome("kayak ");
```

Comment:
Why do you have this space at the end? It does not test word "kayak", but tests word "kayak "